nanomq/apps/broker.c

```diff
@@ -1240,12 +1255,12 @@ broker(conf *nanomq_conf)
 			// nng_free(
```

Comment:
line 1241 `aio_count` should be `conf_total_ctx`

Reply:
rebase branch `bridge/fix` to fix

Reply:
thx